### PR TITLE
Update ClowdEnv with new features

### PR DESCRIPTION
**Pull request by kylape**
- Deploy prometheus
- Deploy unleash
- Use strimzi
- Delete db image (I believe this is deprecated)

### CODE REVIEW

File: bonfire/resources/ephemeral-clowdenvironment.yaml [hunk 1]
```diff
@@ -22,17 +22,16 @@ objects:
...
-        mode: operator
         path: "/metrics"
+        prometheus:
+          deploy: true
+        mode: operator
       kafka:
-        namespace: default
-        clusterName: my-cluster
-        mode: local
+        mode: operator
```
Comment (bsquizz) [inline bonfire/resources/ephemeral-clowdenvironment.yaml:33]: Not quite ready for this yet till the Clowder status PR goes in ;)

Comment (kylape) [reply bonfire/resources/ephemeral-clowdenvironment.yaml:33]: It's merged now :tada:

Comment (bsquizz) [reply bonfire/resources/ephemeral-clowdenvironment.yaml:33]: Testing

Comment (bsquizz) [reply bonfire/resources/ephemeral-clowdenvironment.yaml:33]: Somehow the ClowdEnvironment status still went to 'ready' before a KafkaConnect pod was up .... debugging it.

Comment (bsquizz) [reply bonfire/resources/ephemeral-clowdenvironment.yaml:33]:
```
❯ oc get deployment -n test
NAME                       READY     UP-TO-DATE   AVAILABLE   AGE
env-test-connect-connect   1/1       1            1           3m6s
env-test-entity-operator   1/1       1            1           4m26s
env-test-featureflags      1/1       1            1           6m32s
env-test-minio             1/1       1            1           6m33s
featureflags-db            1/1       1            1           6m32s
```

And on the ClowdEnvironment:
```
  status:
    deployments:
      managedDeployments: 3
      readyDeployments: 3
    generation: 2
    ready: true
    targetNamespace: test
```

Something is not adding up here.

Comment (bsquizz) [reply bonfire/resources/ephemeral-clowdenvironment.yaml:33]: OK, feature flag was not enabled on minikube. Working there now. Will test on ephemeral cluster next.

**Pull request merged**